content/docs/esc/webhooks.md

```diff
@@ -0,0 +1,414 @@
+---
+title_tag: "ESC Webhooks"
+meta_desc: ESC Webhooks allow you to notify external services of events happening within your Pulumi organization. Learn how to create and manage webhooks here.
+title: "Webhooks"
+h1: ESC Webhooks
+meta_image: /images/docs/meta-images/docs-meta.png
+menu:
+  pulumiesc:
+    weight: 9
+aliases:
+- /docs/intro/environment/webhooks/
+- /docs/intro/environments/webhooks/
+- /docs/intro/esc/webhooks/
+- /docs/environment/webhooks/
+- /docs/environments/webhooks/
+- /docs/esc/webhooks/
+---
+
+{{% notes "info" %}}
+ESC Webhooks is a feature available on the Pulumi Team, Enterprise and Business Critical editions.
+To try it out, start a [trial](https://app.pulumi.com/site/trial) now.
+{{% /notes %}}
+
+ESC Webhooks allow you to notify external services of events
+happening within your Pulumi organization. For example,
+you can trigger a notification whenever a new revision of an environment is created.
+Whenever an event occurs, Pulumi will send an HTTP `POST` request to
+all registered webhooks. The webhook can then be used to emit a
```

Comment:
```suggestion
When an event occurs, Pulumi will notify the registered webhook listeners via a HTTP `POST` request with metadata about the event. The webhook can then be used to emit a
```

Reply:
This one is copy-pasted from original page. Should I change both or leave it be? I think we want to stay in sync

Reply:
either way works